bambam.py

```diff
@@ -270,6 +272,8 @@ def run(self):
         self.background = pygame.Surface(self.screen.get_size())
         self.background = self.background.convert()
         self.background.fill((250, 250, 250))
+        if self.args.dark:
+            self.background.fill((0, 0, 0))
```

Comment:
Why fill with white first and then black? Can you put the white fill in an `else:` branch?

Reply:
Added else: branch here https://github.com/porridge/bambam/pull/24/commits/33abab43a1182dae2c7149eacf6fb6b2a9a7624f